exonum-java-binding/core/rust/exonum-java/src/node.rs

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use exonum_time::{time_provider::SystemTimeProvider, TimeServiceFactory};
+use java_bindings::{
+    create_java_vm, create_service_runtime,
+    exonum::{
+        blockchain::{Blockchain, BlockchainBuilder, InstanceCollection},
+        exonum_merkledb::{Database, RocksDB},
+        node::{ApiSender, Node, NodeChannel},
+        runtime::rust::ServiceFactory,
+    },
+    Command, Config, EjbCommand, EjbCommandResult, Executor, InternalConfig, JavaRuntimeProxy,
+};
+
+use std::sync::Arc;
+
+pub fn run_node(command: Command) -> Result<(), failure::Error> {
+    if let EjbCommandResult::EjbRun(config) = command.execute()? {
+        let node = create_node(config)?;
+        node.run()
+    } else {
+        Ok(())
+    }
+}
+
+fn create_node(config: Config) -> Result<Node, failure::Error> {
```

Comment:
Is this testable with testkit?

Reply:
🤔 Not. Should it? Testkit is unable to test our Node creation, it creates its own Node

Reply:
I am asking if the tests for these operations are feasible as they are not currently tested. If not, we'll have to rely on system ones.

Reply:
yeah, we're forced to rely on system ones here. Runtime instantiation will be tested in integration tests though (after testkit is ready)

Reply:
OK, thanks!